owners/src/repo/virtual_repo.js

```diff
@@ -102,6 +102,7 @@ module.exports = class VirtualRepository extends Repository {
    */
   async findOwnersFiles() {
     const ownersFiles = await this.github.searchFilename('OWNERS');
+    ownersFiles.splice(3);
```

Comment:
Magic number?

Reply:
Oof™ testing code, didn't mean to commit that